cmd/iso9660/main.go

```diff
@@ -72,13 +72,14 @@ Usage:
 			panic(err)
 		}
 		
-		if _, err := io.Copy(ff, freader); err != nil {
-			panic(err)
-		}
-
-		if err := ff.Close(); err != nil {
+		_, err = io.Copy(ff, freader)
+		
+		cerr := ff.Close() // With or without an error from Copy, we want to attempt Close.
+		
+		if err != nil { // Panic with Copy's err.
 			panic(err)
+		} else if cerr != nil {
```

Comment:
no need to have the `else if` statement here since panic will interrupt the execution.

Reply:
Sorry, stylistic habit.  You're the owner, so you can nitpick if you want. :)

Btw, great project!  I cross-compiled for arm7 architecture and it helped with a problem I was having.

Reply:
Ah, thanks! I’m glad it was useful for your embedded adventures! :D